docs/001_develop/03_client-capabilities/008_interaction/switch.mdx

```diff
@@ -0,0 +1,198 @@
+```
+Usage
+```typescript
+export function MyComponent() {
+  const handleSwitchChange = (event) => {
+    setSwitchState(!!event.target.checked);
+    console.log({ switchValue })
+  };
+
+  return (
+    <rapid-switch onChange={handleSwitchChange}></rapid-switch>
+  );
+}
+```
+</TabItem>
+<TabItem value="angular">
+
+Declaration
+```html
+<rapid-switch></rapid-switch>
+```
+
+Usage
+```typescript
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+@Component({
+  selector: 'my-root',
+  template: `
+    <rapid-switch
+      [checked]="isSelected"
+      (change)="onSwitchChange($event)"
+    ></rapid-switch>
+  `,
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  imports: [FormsModule],
+})
+export class AppComponent {
+  isSelected: boolean = false;
+  onSwitchChange(event: Event) {
+    const target = event.target as HTMLInputElement;
+    this.isSelected = target.checked;
+    console.log(this.isSelected);
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+## API
+
+Property and attribute binding examples for Genesis Component syntax (closing tag omitted).
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>checked</td>
+      <td><code>boolean</code></td>
+      <td>Sets or retrieves the current state of the component. **Default: `false`**</td>
+<td>
+
+```typescript
+<rapid-switch checked>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>readonly</td>
+      <td><code>boolean</code></td>
+      <td>Configures the component so the user cannot alter the state via the web browser.</td>
+<td>
+
+```typescript
+<rapid-switch readonly>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>disabled</td>
+      <td><code>boolean</code></td>
+      <td>Same functionality as `readonly`, and additionally greys out the component.</td>
+<td>
+
+```typescript
+<rapid-switch disabled>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>form</td>
+      <td><code>string</code></td>
+      <td>Associates this component to a form. Form id needs to be passed. If no Id informed, then it will be associated with the ancestor form.</td>
+<td>
+
+```typescript
+<rapid-switch form="myForm">
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+### Slots
+
+| Slot | Use |
+|---|---|
+| switch | The control element. |
+
+### Parts
+
+| Part    | Description                                       |
+|---------| ------------------------------------------------- |
+| label   | The label text.|
+| checked-indicator | The checked indicator.|
+| checked-message | The text label when the switch is selected.|
+| switch  | The input control.|
+| unchecked-message | The text label when the switch is not selected.|
+
+### Events fired
+| Name    | Description                                                |
+|---------|------------------------------------------------------------|
+| change  | Fires the event when the radio component changes its state |
```

Comment:
Let's put an example of using the change event here, [like this](https://github.com/genesiscommunitysuccess/docs/pull/1895/files#diff-de5bad40a087c6287b3752c04167e66df977b97ed8150a9d0566e108c975053dR213-R236)